imapclient/fetch.go

```diff
@@ -1030,7 +1020,11 @@ func readBodyFldParam(dec *imapwire.Decoder) (map[string]string, error) {
 			if params == nil {
 				params = make(map[string]string)
 			}
-			params[k] = s
+			decoded, err := options.DecodeText(s)
+			if err != nil {
```

Comment:
Can we ignore the error for now, and keep the `// TODO: handle error`?

The reason is that decoding errors shouldn't be fatal, but we have no good way to bubble them up currently.